### DISCUSSION TITLE
Improve the readme File with some useful information

### DISCUSSION THREAD

**Discussion by fineanmol** [fineanmol/hacktoberfest#655] [Ideas]
🐞 **Problem**

Currently Readme file is bit complex to understand for a newbie.

🎯 **Goal**

We can try to make it more readable and add more solutions to it. 

📋  **Steps to solve the problem**

*   Comment below about what you've started working on.
*   Add, commit, push your changes.
*   Submit a pull request and add this in comments - `Addresses #<put issue number here>`
*   Ask for reviews in comments section of pull request.
*   Celebrate your contribution to this project. 🎉

**Comment by kevalsurani15** [comment]
Can I get this task?

**Comment by fineanmol** [comment]
> Can I get this task?

Sure

**Comment by IayushCoderJOD** [comment]
I had started my work thanks

**Comment by mahirjain01** [comment]
hey! I can help , can you please assign it to me?

**Comment by ph1ne4s** [comment]
can I get this task?

**Comment by punyakrit** [comment]
can i get this task

**Comment by AdarshPradhan1508** [comment]
can you  @fineanmol assign this issue to me

**Comment by Abhash-Chakraborty** [comment]
Can you assign me this issue @fineanmol

**Reply by Abhash-Chakraborty** [reply]
please review my changes in PR @fineanmol

**Reply by fineanmol** [reply]
Merged
